src/lib/utils/translations.ts

```diff
@@ -128,6 +128,11 @@ const getRequiredNamespacesForPath = (path: string) => {
   if (path === "run-a-node") {
     requiredNamespaces = [...requiredNamespaces, "page-run-a-node"]
   }
+
+  if (path.startsWith("/wallets")) {
```

Comment:
We need to add this condition in L119 to load the `"learn-quizzes"` namespace as well.
![image](https://github.com/ethereum/ethereum-org-fork/assets/468158/22f55b38-7448-4fa8-8785-c1d08d4e57e8)